src/docs/foundation/accessibility.mdx

```diff
@@ -0,0 +1,59 @@
+---
+name: Accessibility
+menu: Foundation
+route: /foundation/accessibility
+---
+
+# Accessibility
+
+React UI bakes accessibility principles right into its core.
+
+## Touch Friendliness
+
+The active area of interactive elements should be properly sized so that the
+elements can be easily targeted on touch screens. Recommended dimensions may
+vary from platform to platform, however a commonly used size is 7–10 mm.
+
+Default tap target size in React UI is set to **10 mm** and is used by all
```

Comment:
Are we able to somehow read this value from css variable? It would be maintainable.

Reply:
Sure, `--rui-tap-target-size` (and `--rui-form-field-check-tap-target-size` for check fields).

Reply:
I mean, to use value of this CSS varible in the docs, so it changes automatically.

Reply:
I see. It would have to be a React component (AFAIK). I think the idea is related to #224, I'll add it there.